README.md

```diff
@@ -4,20 +4,7 @@ Modules is used to compile avro schemas to typescript interfaces.
 
 # Usage
 ```sh
-npm install avro-to-typescript
-```
-```Typescript
-import {AvroToTypescriptCompiler} from "@degordian/avro-to-typescript";
-const avroToTypescriptCompiler = new AvroToTypescriptCompiler();
-const avroPath = path.resolve( __dirname, `./avscData`);
-const tsCompiledPath = path.resolve( __dirname, `./tsCompiled`);
-
-avroToTypescriptCompiler.tsSchemaPath = path.resolve( tsCompiledPath, `user.ts`);
-avroToTypescriptCompiler.avroSchemaPath = path.resolve( avroPath, `user.avsc`);
-
-avroToTypescriptCompiler.compile().catch( (error: string) => {
-    console.log("Error:", error);
-});
+node dist/src/index.js --compile [ schema-directory ] [ output-directory ]
```

Comment:
I think it should have usage as global and local.
For global usage readme should show something like:
`avro-to-typescript [ schema-directory ] [ output-directory ]`

Reply:
Will look into making global commands